src/styles/utils/emojiBlockquoteStyles/index.android.ts

```diff
@@ -0,0 +1,7 @@
+import type OnlyEmojisBlockquoteLineHeight from './types';
+
+const onlyEmojisBlockquoteLineHeight: OnlyEmojisBlockquoteLineHeight = {
+    lineHeight: 35,
```

Comment:
Just curious, where does 35 come from? Is that used on other platforms or just Android?

Reply:
Sounds good, thanks for the explanation.